### ISSUE TITLE
Infinite scroll fails when scrolling fast

### ISSUE THREAD

**Issue by kevinansfield** [offirgolan/ember-light-table#15]
When scrolling quite fast I've noticed the following behaviour:

1. Scrolling starts and triggers the scroll handler which doesn't pass
2. Scrolling finishes before the throttle timeout
3. No more triggers are fired, no page is loaded unless you scroll back up

I've implemented a quick fix in our app by adding a debounce as well so that we always get a scroll trigger once scrolling stops but there may be a more efficient solution.

```javascript
import Ember from 'ember';
import LightTable from 'ember-light-table/components/light-table';

const {$, run} = Ember;

export default LightTable.extend({
    _setupScrollEvents() {
        $(this.get('touchMoveContainer')).on('touchmove.light-table', run.bind(this, this._scrollHandler, '_touchmoveTimer'));
        $(this.get('scrollContainer')).on('scroll.light-table', run.bind(this, this._scrollHandler, '_scrollTimer'));
        $(this.get('scrollContainer')).on('scroll.light-table', run.bind(this, this._scrollHandler, '_scrollDebounce'));
    },

    _scrollHandler(timer) {
        this.set(timer, run.debounce(this, this._onScroll, 100));
        this.set(timer, run.throttle(this, this._onScroll, 100));
    }
});
```

**Comment by offirgolan** [comment]
I've actually noticed that too in one of my apps. Thanks for bringing this to my attention! Ill take some time to dive in a little deeper as to why this is happening.

**Comment by offirgolan** [comment]
@kevinansfield released with `v1.0.0-beta.1` 😸
